app/src/test/java/com/example/android/architecture/blueprints/todoapp/data/DefaultTaskRepositoryTest.kt

```diff
@@ -101,19 +97,16 @@ class DefaultTaskRepositoryTest {
     @Test
     fun getTasks_requestsAllTasksFromRemoteDataSource() = runTest {
```

Comment:
All `runTest`s in this file should use the `testScope` instead

Reply:
Done